projects/storefrontlib/src/lib/my-account/order/order-history/order-history.component.ts

```diff
@@ -71,15 +72,15 @@ export class OrderHistoryComponent implements OnInit, OnDestroy {
     this.fetchOrders(event);
   }
 
-  pageChange(page: number) {
+  pageChange(page: number): void {
     const event = {
```

Comment:
Can we add a type?

Reply:
Don't see any models with a key of `sortCode`

Reply:
See my previous comment